### PR TITLE
Update Cargo.kamal

### PR DESCRIPTION
**Pull request by Kamal-arc**

### CODE REVIEW

File: query-security-txt/Cargo.toml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [package]
 name = "query-security-txt"
 version = "1.1.1"
-edition = "2021"
```
Comment (nowakwp) [inline query-security-txt/Cargo.toml:4]: Hshha

**Review by nowakwp — COMMENTED** [review]
H5pn5VpUvGbQ8WmA9GBQyfZ3bVVNvoJvpbrwT6dTXmYB

File: query-security-txt/Cargo.toml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 [package]
```
Comment (nowakwp) [inline query-security-txt/Cargo.toml:1]: H5pn5VpUvGbQ8WmA9GBQyfZ3bVVNvoJvpbrwT6dTXmYB

**Review by nowakwp — COMMENTED** [review]
H5pn5VpUvGbQ8WmA9GBQyfZ3bVVNvoJvpbrwT6dTXmYB

**Review by nowakwp — COMMENTED** [review]
H5pn5VpUvGbQ8WmA9GBQyfZ3bVVNvoJvpbrwT6dTXmYB

**Comment by nowakwp** [comment]
> H5pn5VpUvGbQ8WmA9GBQyfZ3bVVNvoJvpbrwT6dTXmYB



> H5pn5VpUvGbQ8WmA9GBQyfZ3bVVNvoJvpbrwT6dTXmYB

**Review by nowakwp — COMMENTED** [review]
H5pn5VpUvGbQ8WmA9GBQyfZ3bVVNvoJvpbrwT6dTXmYB

**Review by Sturgeontara06 — CHANGES_REQUESTED** [review]
Don't have a clue

**Review by Elgayar777 — APPROVED** [review]

**Review by Elgayar777 — COMMENTED** [review]
.

**Review by Sturgeontara06 — APPROVED** [review]

**Review by Sturgeontara06 — COMMENTED** [review]
Ok

**Comment by JanKortes212** [comment]
What the adres is this?

**Pull request closed without merging**